solvers/svrg.py

```diff
@@ -0,0 +1,45 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    from tick.linear_model import ModelLogReg
+    from tick.prox import ProxL2Sq
+    from tick.solver import SVRG
+
+
+class Solver(BaseSolver):
+    name = 'svrg-tick'
+
+    install_cmd = 'conda'
+    requirements = [
+        'pip:git+https://github.com/X-DataInitiative/tick.git'
+    ]
+    references = [
+        'Bacry, Emmanuel, et al. "Tick: a Python library for statistical learning, '
+        'with a particular emphasis on time-dependent modelling." '
+        'arXiv preprint arXiv:1707.03003 (2017).'
+    ]
+
+    parameters = {
+        'step_type': [
+            'bb',
+            'fixed',
+        ]
+    }
+
+    def set_objective(self, X, y, lmbd):
+        self.X, self.y, self.lmbd = X, y, lmbd
+        # Reponse vector used for tick logreg must be {1.0, -1.0}
+        self.y[y == 0] = -1.0
```

Comment:
I would instanciate the model here as this is not really part of the solver and the heavy initial computations (step size) are done in the `run`

```suggestion
        model = ModelLogReg(fit_intercept=fit_intercept).fit(X, y)
        # Scaled with 1/n_samples as logistic objective is taken with mean
        prox = ProxL2Sq(lmbd / X.shape[0])
```

Reply:
the labels are already preprocess in the datasets to be `-1, 1`, see check in objective